spec/openapi.yaml

```diff
@@ -6402,6 +6458,70 @@ components:
             next_token:
               type: string
               description: The pagination token of the next set of results.
+    PatchLabelMyOwnNumber:
+      type: object
+      title: PatchLabelMyOwnNumber
+      required:
+        - label
+      properties:
+        label:
+          type: string
+          description: Label need to be updated
+          example: "My Number"
+    UpdatedLabelAddressItem:
```

Comment:
Why we need a new? Should we re-use the already-old ?

Reply:
because don't have existing to reuse